components/resources/demo/shared/build.gradle.kts

```diff
@@ -1,13 +1,21 @@
+import org.jetbrains.kotlin.gradle.ExperimentalKotlinGradlePluginApi
+
 plugins {
     kotlin("multiplatform")
     id("com.android.library")
     id("org.jetbrains.compose")
 }
 
-version = "1.0-SNAPSHOT"
-
 kotlin {
-    android()
+    @OptIn(ExperimentalKotlinGradlePluginApi::class)
+    targetHierarchy.default()
+    androidTarget {
+        compilations.all {
+            kotlinOptions {
+                jvmTarget = "11"
```

Comment:
Shouldn't we use `jvmToolchain(11)` instead?

Reply:
the jvmToolchain is not my favorite thing to use. we want to set java version only for the android target. so it seems easier to use explicit jvmTarget.